zhplot/matplotlib.py

```
@@ -0,0 +1,31 @@
+import os
```

Ruff (0.8.0)

`os` imported but unused

Remove unused import: `os`

(F401)